JS_HW/task_7.js

```diff
@@ -0,0 +1,37 @@
+// Написать функцию, которая принимает букву, если буква является большой, функция возвращает ‘Оууу май, большая буква!’. В ином случае ‘Нет уж, маленькие буквы - скучно’
+
+// Пример: 
+// checkLetterCase(‘A’) // ‘Оууу май, большая буква!’
+// checkLetterCase(‘s) //  ‘Нет уж, маленькие буквы - скучно’
+
+
+
+
+
+
+
+
+function checkLetterCase (stringValue) {
+    
+    if ((stringValue.length !== 1) && (typeof stringValue !== 'string')) {
+        console.log ('Data entered incorrectly')
+
+        return
+    }
+
+    if (stringValue.length === 1) {
```

Comment:
redundant comparison. In a code above you have already handled the case when user pass more or less then 1 letter